core/metacling/src/TCling.cxx

```diff
@@ -1258,9 +1290,37 @@ TCling::TCling(const char *name, const char *title)
       // Load libc and stl first.
       LoadModules({"libc", "stl"}, *fInterpreter);
 
-      LoadModules({"ROOT_Foundation_C", "ROOT_Config", "ROOT_Foundation_Stage1_NoRTTI", "Core", "RIO"}, *fInterpreter);
-      if (!fromRootCling)
-         LoadModules({"TreePlayer", "ROOTVecOps"}, *fInterpreter);
+      // Load core modules
+      std::vector<std::string> CoreModules = {"ROOT_Foundation_C","ROOT_Config","ROOT_Foundation_Stage1_NoRTTI", "Core", "RIO"};
+      // Exclude buggy modules
+      // FIXME: Reducing those will let us be less dependent on rootmap files
+      std::vector<std::string> ExcludeModules = {"Rtools", "RSQLite", "Rint", "RInterface", "RMVA"};
+
+      LoadModules(CoreModules, *fInterpreter);
+
+      // Otherwise module conflicts between rootcling and root
```

Comment:
"Otherwise" what? Could you extend this to make it easier to understand what you're saying here?